src/pages/host/Metadata.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { ApolloError } from "apollo-client";
+import { StyledLink } from "components/styles";
+import { HostMetaDataCard } from "pages/host/HostMetaDataCard";
+import { HostQuery } from "gql/generated/types";
+import { getUiUrl } from "utils/getEnvironmentVariables";
+import { getDateCopy } from "utils/string";
+import { P2 } from "components/Typography";
+
+export const Metadata: React.FC<{
+  loading: boolean;
+  data: HostQuery;
+  error: ApolloError;
+}> = ({ loading, data, error }) => {
+  const host = data?.host ?? null;
+
+  const hostUrl = host?.hostUrl;
+  const distroId = host?.distroId;
+  const startedBy = host?.startedBy;
+  const provider = host?.provider;
+  const user = host?.user;
+  const lastCommunicationTime = host?.lastCommunicationTime;
+
+  const runningTask = host?.runningTask;
+  const runningTaskId = runningTask?.id;
+  const runningTaskName = runningTask?.name;
+  const sshCommand = `ssh ${user}@${hostUrl}`;
+
+  const taskLink = `${getUiUrl()}/task/${runningTaskId}`;
+  const distroLink = `${getUiUrl()}/distros##${distroId}`;
+
+  return (
+    <HostMetaDataCard error={error} loading={loading}>
+      <P2>User: {user}</P2>
+      <P2>Host Name: {hostUrl}</P2>
+      <P2>SSH Command: {sshCommand}</P2>
```

Comment:
There are 2 ways i can think of to clearly display the ssh command and make it usable you can 
A: Use a leafygreen [code component ](https://github.com/mongodb/leafygreen-ui/tree/master/packages/code) It will supply a formatted code component with a copy button.

B: Use the `pre` tag and format it to show the data. Heres a [link](https://github.com/evergreen-ci/spruce/blob/04dc0f31244de7c554bbb7fffac085ce02007438/src/pages/preferences/preferencesTabs/cliTab/VerifyCard.tsx#L51) to it in our code base
I used a combination of both on the [preferences/cli page](https://spruce.mongodb.com/preferences/cli)

Reply:
![image](https://user-images.githubusercontent.com/13104717/89548813-7d650600-d7d5-11ea-9503-672dee7ae31d.png)
I proposed this in the design channel and waiting for feedback.